python/django/python-hello-world/src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==2.2.1
```

Comment:
please add a brief readme file that focuses on:
1) How to build
2) How to debug
3) How to deploy
All could be links to existing cloud.google.com documentation.
It should also show the sample app "architecture" (maybe worth highlighting django involvement in the sample)

You can look at existing hello-world python sample readme for inspiration/

Reply:
I added a readme file, and this readme file is basically the same with the readme file in the existing hello-world python sample, except the diagram showing the structure of the template, and the description of the content of the template.